### ISSUE TITLE
Video stream acting weirdly 

### ISSUE THREAD

**Issue by sansyrox** [mexili/incogly-backend#3]
Videos are sometimes visible to only a few participants in the call.

**Comment by nimit2801** [comment]
Hey, @sansyrox is there a debug log for the following error, more info would help to solve this bug.
Thank You!

**Comment by sansyrox** [comment]
@nimit2801 , yes it is an issue with video codecs and the wavering webrtc servers in our code. So we will be changing the native socketio rooms to peerjs rooms.

**Comment by sansyrox** [comment]
@nimit2801 , lmk if you'd be interested in its implementation.

**Comment by nimit2801** [comment]
I've worked with socket.io a bit but don't have enough experience with peerjs. Sorry for the late reply got busy with some projects.
Would love to check this project out on weekends. 
The problems I can infer from this issue is:
1. The issue is with the library socket.io and want to revamp the code base with peerjs
2. peerjs would solve this by introducing its own simple webrtc solution
